core/client/verify.go

```diff
@@ -38,26 +38,26 @@ var (
 	ErrNilProof = errors.New("nil proof")
 )
 
-// Verifier is a client helper library for verifying request and responses.
-type Verifier struct {
-	vrf         vrf.PublicKey
-	mapVerifier *tclient.MapVerifier
-	logVerifier *tclient.LogVerifier
+// RealVerifier is a client helper library for verifying request and responses.
```

Comment:
Maybe you could make a note that this implements `Verifier`? That would explain the seemingly unclear `Real` bit.

Reply:
Done